web/src/features/fbaCalculator/components/FBAInputGrid.tsx

```diff
@@ -417,6 +440,12 @@ const FBAInputGrid = (props: FBAInputGridProps) => {
             </TableHead>
             <TableBody data-testid="fba-table-body">
               {sortedRows.map((row, ri) => {
+                const criticalHoursString4000 = formatCriticalHoursAsString(
```

Comment:
Might be easier to read to just call the formatting function where critical hours is displayed